### ISSUE TITLE
Access Denied on mounted folder

### ISSUE THREAD

**Issue by tonyhxh** [waldekmastykarz/docker-spfx#17]
Hi, Waldek:

I probably have an unique issue. I tried to use your image, however I can't share my drive directly in docker settings due to enterprise policy. I suspected GPO disabled SMB V1. 

Through some research, I tried to manually map my D drive to docker host. 
1. Share D drive
   net share D=D:\ /grant:"{domain}\{user}",FULL /CACHE:None

2. Map D drive to docker host, and force SMB V2
docker run --rm -it --privileged --userns=host --pid=host --network=host --ipc=host --uts=host osexp2000/nsenter-dockerd sh

/# mnt=/var/lib/docker/D
/# [ -e $mnt ] || mkdir $mnt
/# mount -t cifs //192.168.65.2/D $mnt -o user={user}, dom={domain}, vers=2.0

3. Run container with mapped folder
docker run -it --rm --name SPXProjects -v /var/lib/docker/D/SPXProjects/1.6.0:/usr/app/spfx -p 5432:5432 -p 4321:4321 -p 35729:35729 waldekm/spfx:1.6.0
spfx@1a1f69103ec0:/usr/app/spfx$ ls
test.txt
spfx@1a1f69103ec0:/usr/app/spfx$ mkdir test
mkdir: cannot create directory 'test': Permission denied
spfx@1a1f69103ec0:/usr/app/spfx$

As you can see, in the container, I can view the folder, but can't change anything inside it. I am wondering if there is a way to elevate user spfx's permission. 

When I mapped a folder to a container based on image ubuntu, I don't see any issue. 
docker run -it --rm -v /var/lib/docker/D/Docker/learningLinux:/learningLinux ubuntu
root@2cc90e2287f0:/# cd learningLinux
root@2cc90e2287f0:/learningLinux# ls
test1.txt  test2.txt  test3.txt  test4.txt
root@2cc90e2287f0:/learningLinux# mkdir test
root@2cc90e2287f0:/learningLinux# ls
test  test1.txt  test2.txt  test3.txt  test4.txt
root@2cc90e2287f0:/learningLinux#

Thanks
Tony Huang

**Comment by waldekmastykarz** [comment]
Sorry, I'm afraid I don't have any suggestions for you. Is this the only image you have encountered this problem with?

**Comment by tonyhxh** [comment]
Yes. I spent way too much time to try to use your docker image, but I couldn't share my drive with docker on windows 10. In corporate environment, I suspect it will happen a lot. So far I still can't figure out why docker can't mount the drive. 

I tracked the docker settings through process monitor. It will fail at "nc -zv -w1 10.0.75.1 445", which means my host's IP 10.0.75.1 is not listening 445. Firewall looks fine, so I don't know what I can do. 

I do have a workaround, which is mounting the drive through host.docker.internal (192.168.65.2:445) manually. When I mapped a folder in the drive to your image, it seems the user spfx only can have read permission to the folder. When I mapped the folder to some other images I pull from docker hub, it seems to be fine. Of course, the users usually are root. I guess I am stuck.

**Comment by tonyhxh** [comment]
Thanks, I am good now.
